### ISSUE TITLE
README.md import/export modes constants

### ISSUE THREAD

**Issue by I-is-as-I-does** [dwgebler/doclite#2]
Hi!
A tiny thing.
In readme.md, 'Import and export data' [section](https://github.com/dwgebler/doclite#import-and-export-data):
These constants throw exceptions:
```
Database::EXPORT_COLLECTIONS;
Database::EXPORT_DOCUMENTS;
```
These work fine:
```
Database::MODE_EXPORT_COLLECTIONS;
Database::MODE_EXPORT_DOCUMENTS;
```
Anyway, thanks a lot for DocLite.

**Comment by dwgebler** [comment]
Cool, will update documentation next commit, glad you're enjoying DocLite...I would add import/export is probably the most underdeveloped feature and one which needs improving in future, it should be fine using JSON format and for fairly small databases but beyond that it might be less reliable (at least for import).

**Comment by dwgebler** [comment]
Fixed in v1.1.0.
